app/angular/src/client/preview/angular-beta/StorybookWrapperComponent.ts

```diff
@@ -42,6 +42,10 @@ export const createStorybookWrapperComponent = (
   styles: string[],
   initialProps?: ICollection
 ): Type<any> => {
+  // In ivy, a '' selector is not allowed, therefore we need to just set it to anything if
+  // storyComponent was not provided.
+  const viewChildSelector = storyComponent ?? '__storybook-noop';
```

Comment:
Can we have this fix in 6.2-alpha? Somehow someway of our developers face an empty view child selector error that breaks storybook. This change applied manually to storybook dist fixes it.
Looks like somehow angular in storybook can run in ivy mode but I can't reproduce this :(